### PR TITLE
Subaru preglobal update

### PR DESCRIPTION
**Pull request by martinl**
This PR adds Subaru preglobal gear values and disables automatic checksum generation for preglobal DBC files. Prerequisite for merging https://github.com/commaai/openpilot/pull/1489

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Can you clean up all the NEW_SIGNAL and NEW_MSG if we don't know what they are?

File: subaru_outback_2015_eyesight.dbc [hunk 1]
```diff
@@ -369,3 +371,6 @@ CM_ SG_ 880 Steering_Voltage_Flat "receives later than 371";
 CM_ SG_ 880 NEW_SIGNAL_1 "0 in 2017";
 CM_ SG_ 880 NEW_SIGNAL_4_2017 "1 in 2017";
 CM_ SG_ 880 NEW_SIGNAL_5_2017 "1 in 2017";
+CM_ SG_ 1745 Units "0 = Metric, 1 = Imperial";
+CM_ SG_ 1745 Units2 "0 = Metric, 1 = Imperial";
```
Comment (adeebshihadeh) [inline subaru_outback_2015_eyesight.dbc:375]: I think values would be more appropriate than a comment here

**Comment by martinl** [comment]
I removed unknown signals, reordered known signals and defined Units values. Some ES_CruiseThrottle signal names changed, so I will update carstate in preglobal PR accordingly

File: subaru_forester_2017.dbc [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+BO_ 355 ES_DashStatus: 8 XXX
+ SG_ Not_Ready_Startup : 4|2@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_On : 16|1@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_Set_Speed : 24|8@1+ (1,0) [0|255] "" XXX
+ SG_ Counter : 40|3@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_Activated : 54|1@1+ (1,0) [0|1] "" XXX
+
+
+
+
```
Comment (adeebshihadeh) [inline subaru_forester_2017.dbc:233]: one line

Comment (martinl) [reply subaru_forester_2017.dbc]: removed empty lines

File: subaru_outback_2015_eyesight.dbc (hunk 1 above)
Comment (martinl) [reply subaru_outback_2015_eyesight.dbc]: Changed comments to values

**Pull request merged**